Task 8/script.js

```diff
@@ -5,3 +5,28 @@ subtraction(a, b) - priima du skaičius ir grąžina jų skirtumą.
 multiplication(a, b) - priima du skaičius ir grąžina jų daugybos rezultatą;
 division(a, b) - priima du skaičius ir grąžina jų dalybos rezultatą;
 ------------------------------------------------------------------------------------ */
+
+function Calculator (a, b) {
```

Comment:
Minor thing, but intention was for every method to accept arguments `a` and `b` so arguments from `Calculator` could be deleted and just all the methods changed to accept those arguments.